datastore/cloud-client/snippets.py

```diff
@@ -110,10 +229,13 @@ def properties(client):
 
 
 def array_value(client):
-    key = client.key("Task")
     # [START datastore_array_value]
+    key = client.key("Task")
     task = datastore.Entity(key)
     task.update({"tags": ["fun", "programming"], "collaborators": ["alice", "bob"]})
+
+    # To access array values:
+    task["tags"]  # ["fun", "programming"]
```

Comment:
this ticks out as a bit strange. Do we have a read array sample? if not including here may be fine. What do other langs do? Should they also be updated?

Reply:
Resolving this with additional tickets, per our conversation.